src/reservationPage.js

```diff
@@ -0,0 +1,128 @@
+import { createReservation } from './involvementReservationAPI';
```

Comment:
- [OPTIONAL] For this page, you could improve the design a little bit:
- The pop-up and navbar are closer, please fix it.
- The submit button is a little hard to see and closer to the input, please fix it too.
![image](https://user-images.githubusercontent.com/20563252/132966043-becb12df-6f84-4c36-9f13-e1f8de2d9d5f.png)

Reply:
* Done 👍